src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -218,7 +221,9 @@
                           ; this is here primarily for old users without :firstName/:lastName fields
                           "Profile"
                           name)))
-              success? ; partial profile case
+              (and success? (>= (int (:isRegistrationComplete parsed-values)) 1)) ; partial profile case
```

Comment:
it's possible - but I think very unlikely - that the value of isRegistrationComplete will be "true" - this is for anyone who registered before 1/11/2016, and who has not visited the site since then. Might be safest to check not-nil here instead.